cumulusci/core/keychain/tests/test_encrypted_file_project_keychain.py

```diff
@@ -61,7 +76,33 @@ def test_set_and_get_org__global(self, keychain, org_config):
         assert list(keychain.orgs.keys()) == ["test"]
         assert keychain.get_org("test").config == org_config.config
 
-    def test_set_and_get_org__universal_config(self, key, org_config):
+    def test_get_org__not_found(self, keychain):
+        org_name = "mythical"
+        error_message = f"Org with name '{org_name}' does not exist."
+        with pytest.raises(OrgNotFound, match=error_message):
+            keychain.get_org(org_name)
+
+    def test_set_org__no_key_should_save_to_unencrypted_file(
+        self, keychain, org_config
+    ):
+        keychain.key = None
+        keychain.set_org(org_config)
+
+        filepath = Path(keychain.project_local_dir, "test.org")
+        with open(filepath, "rb") as f:
+            assert pickle.load(f) == org_config.config
+
+    @mock.patch("cumulusci.core.keychain.encrypted_file_project_keychain.open")
+    def test_save_org_when_no_project_local_dir_present(
+        self, mock_open, keychain, org_config
+    ):
+        with mock.patch.object(EncryptedFileProjectKeychain, "project_local_dir", None):
+            keychain._save_org("alias", org_config, global_org=False)
+        assert mock_open.call_count == 0
```

Comment:
Is this the right assertion given that we're using os.open?

Reply:
We use `os.open` to get a file descriptor which we pass to the built-in `open()` function.